tmpl/conversions.tmpl

```diff
@@ -1,5 +1,5 @@
 //glow:keepspace
-// Glow automatically generated OpenGL binding: http://github.com/go-gl/glow
+// Code generated by glow (http://github.com/go-gl/glow). DO NOT EDIT.
```

Comment:
Please update the URL scheme to https. Here, and below.

```suggestion
// Code generated by glow (https://github.com/go-gl/glow). DO NOT EDIT.
```

Reply:
Done